### DISCUSSION TITLE
The game window

### DISCUSSION THREAD

**Discussion by Buduf** [Buduf/PTI-Ear-Trainer#3] [Ideas]
Here's my idea for a UI to the Game Window.
In this Window the user have to guess the played interval. This can be replayed with the middle play, triangle button. The buttons bellow are the intervals the user needs to guess from. There's a label at the top, which indicates the number of the guesses. Also there's a label at the bottom displaying the time remaining/elapsed (depending on the game mode).
After a guess have been made, a custom user control shows the exact sounds in a musical notation. (Will be detailed in another discussion)

![image](https://user-images.githubusercontent.com/55985313/196771318-0c0d623c-2e46-43e8-bb19-2a312904060f.png)
What do you think? Have I left somethnig?

**Comment by Falesz** [comment]
Nice initial idea! Depending on game mode, we will need more controls though. For example, during practice there should be a button to skip the interval or generate a new one in case the user can't or can't be bothered to identify the interval.

**Comment by Buduf** [comment]
I think game mode specific controls could be created dynamically. Because: 
- We can't derive from windows.
- Multiple windows for each game mode would  increase code redundancy.
